resources/knative-build/templates/tests/test-build.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Chart.Name }}-tests
+  labels:
+    helm-chart-test: "true"
+rules:
+- apiGroups: ["build.knative.dev"]
+  resources: ["builds"]
+  verbs: ["*"]
+- apiGroups: 
+  - ""
+  resources:
+  - pods/log
+  - pods
+  verbs:
+  - list
+  - get
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Chart.Name }}-tests
+  namespace: knative-build
+  labels:
+    helm-chart-test: "true"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Chart.Name }}-tests
+  namespace: knative-build
+  labels:
+    helm-chart-test: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Chart.Name }}-tests
+subjects:
+- kind: ServiceAccount
+  name: {{ .Chart.Name }}-tests
+  namespace: knative-build
+---
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
```

Comment:
Please align it to the unification efforts in https://github.com/kyma-project/kyma/pull/4958

Reply:
fixed.